CONTRIBUTING.md

```diff
@@ -1,91 +1,128 @@
+### Making a new cheat sheet category (mods, expansions, etc)
 
-### If you're making a new cheat sheet category (mods, expansions, etc)
+Follow these steps:
 
-- In the [cheat-sheets](./src/app/cheat-sheets/) folder add a new category folder using the `deniszholob.angular-files-generator` extension to generate a new `Module Component`
-- Follow the other category examples to create new routes, update the html etc..
-- Update the [cheat-sheets.module](./src/app/cheat-sheets/cheat-sheets.module.ts)
-- Update the [dav.data](./src/app/layout/nav/nav.data.ts)
-- Add new route to the [sitemap.xml](./src/sitemap.xml)
-- Add cheat sheets as needed following the guide above
+1. In the [`cheat-sheets` folder](./src/app/cheat-sheets/) use the `deniszholob.angular-files-generator` extension to generate a new `Module Component`.
+2. Follow the other category examples to create new routes, update the HTML and so on.
+3. Update the [`cheat-sheets.module.ts`](./src/app/cheat-sheets/cheat-sheets.module.ts) file.
+4. Update the [`nav.data.ts`](./src/app/layout/nav/nav.data.ts) file.
+5. Add new route to the [`sitemap.xml`](./src/sitemap.xml) file.
+6. Add cheat sheets, if needed, by following the steps listed in the _Making a new cheat sheet_ section above.
 
-## Steps to follow when your work is ready:
+## Before opening a Pull Request
 
-When your work is done:
+When your work is done, please:
 
-1. Run `npm run build`.
-2. After a successful build, make a commit and push your changes. If you're fixing a existing issue: be sure to link to that issue in the git commit message, like so: `Closes #issueNumberThatGetsFixed`.
-3. Create a new Pull Request.
-4. Write a good description of the changes this pull-request will make.
-5. You must provide screenshots if there is a visual change.
+1. If you're making visual changes to the cheatsheets: run the preview server with `npm run start` and make screenshots of your changes.
+2. Run `npm run build`.
+3. After a successful build, commit and then push your changes to your fork. If you're fixing an Issue on GitHub: link to that issue in your Git commit message, for example: `Closes #issueNumberThatGetsFixed`.
+4. Create a new Pull Request, and follow the steps in the Pull Request template.
 
 ## Debug
 
-- If you get an error with precommit hooks (Mac/Linux), see: https://stackoverflow.com/a/72279243
+If you get an error with precommit hooks on Mac/Linux, see this [discussion on Stack Overflow](https://stackoverflow.com/a/72279243).
 
-## Generating PDF
+## Generating a PDF
 
-- Use Firefox print
-- Remove
-  - margins
-  - headers/footers
-- Keep backgrounds
-- Scale to 90%
-- Remove last page as its empty
+1. Use Firefox, to "print" the page to PDF.
+2. Remove the margins, headers and footers.
+3. But keep the backgrounds.
+4. Scale to `90%`.
+5. Remove the last page, because it's empty.
```

Comment:
```suggestion
5. Remove the last page if it's empty.
```
Not sure if thats the case, dont want some1 blindly removing last page if it has content XD